src/part-2-intro.md

```diff
@@ -1,4 +1,4 @@
-# Part 2: How rustc works
+# Part 2: How `rustc` Works
```

Comment:
Hmm... I think high-level compiler arch is actually a better name (also in the about this guode section).

Reply:
978d894938f0d0990df1b82c4b72ee77381625f6